test/sharness/t0140-swarm.sh

```diff
@@ -101,11 +101,12 @@ test_launch_ipfs_daemon
 
 test_expect_success "Addresses.NoAnnounce affects addresses from Announce and AppendAnnounce" '
   ipfs swarm addrs local >actual &&
-  grep -v "/ip4/1.2.3.4/tcp/1234" actual &&
-  grep -v "/ip4/10.20.30.40/tcp/4321" actual &&
+  cat actual && # confirm the bug
+  test_should_not_contain "/ip4/1.2.3.4/tcp/1234" actual &&
+  test_should_not_contain "/ip4/10.20.30.40/tcp/4321" actual &&
   ipfs id -f"<addrs>" | xargs -n1 echo >actual &&
-  grep -v "/ip4/1.2.3.4/tcp/1234" actual &&
-  grep -v "//ip4/10.20.30.40/tcp/4321" actual
+  test_should_not_contain "/ip4/1.2.3.4/tcp/1234" actual &&
+  test_should_not_contain "//ip4/10.20.30.40/tcp/4321" actual
```

Comment:
that `//` looks like a typo

Reply:
Nice catch! Yes, it looks like copy&paste bug of sorts :see_no_evil: 
Anyway, the test was broken because after https://github.com/ipfs/kubo/pull/8177 we should test both addresses added via `Announce` and `AppendAnnounce` (so we need two specimens):

```diff
-noAnnounceCfg='["/ip4/1.2.3.4/tcp/1234"]'
+noAnnounceCfg='["/ip4/1.2.3.4/tcp/1234", "/ip4/10.20.30.40/tcp/4321"]'
```

I've pushed fix in 038e56d4130d85ef1423df1db26f8b6f96e891f2 and should be green now :crossed_fingers: